rollingpin/args.py

```diff
@@ -2,6 +2,8 @@
 import os
 import sys
 
+import commands
```

Comment:
nit: it's best not to use implicit relative imports like this as they're very confusing about where the module actually lives (e.g. this would be pretty confusing if a top-level `commands` module were installed)